vavr/src/main/java/io/vavr/PartialFunction.java

```diff
@@ -42,6 +42,33 @@
     long serialVersionUID = 1L;
 
     /**
+     * Unlifts a {@code totalFunction} that returns an {@code Option} result into a partial function.
+     * The total function should be side effect free because it might be invoked twice: when checking if the
```

Comment:
👍🏼

Reply:
Thanks for your review! I will have a look into the issue you mentioned ... buy the way: the Java 9 compilation step within the Travis build seems to be flickering?

Reply:
> Java 9 compilation step within the Travis build seems to be flickering?

yes, will take a look...